distribution/build.gradle

```diff
@@ -238,6 +238,34 @@ project.rootProject.subprojects.findAll { it.parent.path == ':modules' }.each {
   restTestExpansions['expected.modules.count'] += 1
 }
 
+// copy all sandbox modules if the distribution is a snapshot
+if (VersionProperties.isOpenSearchSnapshot()) {
+  Properties sysProps = System.getProperties();
+  // setting this property to false will exclude the sandbox modules from the distribution
+  final String enableSandbox = sysProps.getProperty("sandbox.enabled", "true");
```

Comment:
Since sandbox modules are included by default in snapshot builds, does that mean they have a hard requirement to pass bwc checks from day 1, assuming bwc runs (or will run) as part of gradle check, which do run on snapshot artifacts. Pls correct if any of my assumptions here are wrong. :)

Reply:
Yes - if we want to create a distribution with any new feature, the checks (including all bwc) need to pass.